backend/src/core/models/campaign.ts

```diff
@@ -104,6 +104,12 @@ export class Campaign extends Model<Campaign> {
   })
   demoMessageLimit!: number
 
+  @Column({
+    type: DataType.BOOLEAN,
+    defaultValue: false,
+  })
+  shouldBccToMe!: boolean
```

Comment:
side note for schema design consideration: although not immediately imperative, this is an e-mail campaign unique field. in strict DAO sense, this field should be in an extension table unique to e-mail campaigns.

Reply:
I'd prefer we keep thing simple first as there's not enough benefits in doing this